Makefile

```diff
@@ -117,7 +117,7 @@ update-mockgen:
 
 update-proto-plugins:
 	@printf $(COLOR) "Install/update proto plugins..."
-	@go install github.com/temporalio/gogo-protobuf/protoc-gen-gogoslick@latest
```

Comment:
I released new version of plugin. Please revert this change and rebase after #3300 is merged.

Reply:
Sure. I reverted the commit and wait for the PR to merge.